src/Chirp.CLI.Client/Program.cs

```diff
@@ -1,18 +1,17 @@
 ﻿namespace Chirp.CLI.Client;
 
-
-using System.Diagnostics;
-using System.Text.RegularExpressions;
+using Chirp.CSVDB;
+using System.Globalization;
 
 public class Program
 {
 
-    static string filePath = @"../../data/Chirp.CLI/chirp_cli_db.csv";
-    static string userName = Environment.UserName;
-    static long timestamp = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
-    static int count = 0;
-        
-    static void Main(string[] args)
+    string filePath = @"../../data/Chirp.CLI/chirp_cli_db.csv";
+    string userName = Environment.UserName;
+    long timestamp = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
+    int count = 0;
+    
+    public void Main(string[] args)
```

Comment:
If we have the main function in `App.cs`, also having a main function in `Program.cs` might be confusing in the future. Furthermore, i recommend we change the class/file name to something else than `Program.cs`.